README.md

```diff
@@ -10,6 +10,36 @@ This is the git repository for [openSUSE:Tools/obs-service-tar_scm](https://buil
 
 The service can be used in combination with other services like [download_files](https://github.com/openSUSE/obs-service-download_files), [extract_file](https://github.com/openSUSE/obs-service-extract_file), [recompress](https://github.com/openSUSE/obs-service-recompress) or [set_version](https://github.com/openSUSE/obs-service-set_version) e.g. within the [GIT integration](https://en.opensuse.org/openSUSE:Build_Service_Concept_SourceService#Example_2:_GIT_integration) workflow.
 
+## Archive Formats
+
+### obscpio
+The obscpio archive format is the new OBS specific archive format which used and generated by the [OBS Delta Store](http://openbuildservice.org/help/manuals/obs-reference-guide/cha.obs.architecture.html#delta_store) to store SCM changes in an incremental way. 
+It is used by the obs_scm service. A file <package>.obsinfo including useful information from your SCM system is generated, so you can access this data at buildtime.
+
+### tar
+The tar archive format is used to create a tarball and it is used by the services "tar" and "tar_scm"
```

Comment:
Similar thing here - if a file format is "used by" X, that could mean either X is generating files in that format *or* X is consuming files in that format which were generated by something else.